README.rst

```diff
@@ -43,12 +43,38 @@ How to use
 
 Plugin contains three fixtures
 
-* **redisdb** - it's a client fixture that has functional scope. After each test, it cleans Redis database for more reliable tests.
+* **redisdb** - This is a redis client fixture. It constructs a redis client and cleans redis database after the test.
+    It relies on redis_proc fixture, and as such the redis process is started at the very beginning of the first test
+    using this fixture, and stopped after the last test finishes.
 * **redis_proc** - session scoped fixture, that starts Redis instance at it's first use and stops at the end of the tests.
 * **redis_nooproc** - a nooprocess fixture, that's connecting to already running redis
 
 Simply include one of these fixtures into your tests fixture list.
 
+.. code-block:: python
+
+    #
+    def test_redis(redisdb):
```

Comment:
I'm still not sure to get how a different client can be instantiated, from what I understand the redis client is hidden in `redisdb `.

Reply:
@cglacet just different client, or different instance alltogether?